app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class VotesController < ApplicationController
+  def index
+    review = Review.find(params[:review_id])
+    total = review.votes.map { |v| v[:value] }
+    @score = total.reduce(:+)
+    respond_to do |format|
+      format.json { render json: @score }
+    end
+  end
```

Comment:
Not necessary this very moment, but if you want to follow best practices, then I would figure out how to get all this logic into the Model instead of the controller, so that when you call `Review.score` or something like that it gives you the total score. Add an issue to github as a refactor.